spec/features/consumer/shopping/cart_spec.rb

```diff
@@ -56,6 +56,7 @@
       let(:variant) { product_tax.variants.first }
 
       before do
+        Spree::Config.set allow_backorders: false
```

Comment:
Would we need to restore this to its original value after this spec has run? I assume that it'll maintain its value after the specs in this block.